### DISCUSSION TITLE
Direct sharing of links in binary files

### DISCUSSION THREAD

**Discussion by MKdays** [community/community#23202] [New to GitHub]
<p>Most people are not interested in the source code and will need a binary file.</p>
<p>Is link sharing allowed only for Binary files in someone’s repository?</p>
<p>Because from GitHub’s point of view, this is just increasing traffic. There’s no PR for GitHub at all.</p>
<p>Is there a good way to effectively provide only binaries to end-users?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>You may want to check out our Releases feature.</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/github/administering-a-repository/about-releases" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/administering-a-repository/about-releases" target="_blank" rel="noopener">About releases - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/administering-a-repository/about-releases" target="_blank" rel="noopener">//docs.github.com/en/github/administering-a-repository/about-releases</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by MKdays** [comment]
<p>Does this mean that it is okay to directly share links to Binary files from other sites or the Internet?</p>
<p>For example,<br>
<a href="https://github.com/MKdays/PyKooler/releases/latest/download/app.zip" class="onebox" target="_blank" rel="noopener nofollow ugc">https://github.com/MKdays/PyKooler/releases/latest/download/app.zip</a></p>

**Comment by canuckjacq** [comment]
<p>Yes - we have a guide to <a href="https://docs.github.com/en/github/administering-a-repository/linking-to-releases#:~:text=You%20can%20share%20every%20release,click%20Releases%20or%20Latest%20release.">linking to your GitHub Releases</a> for more information.</p>
